### DISCUSSION TITLE
how i can use the search box for filter by groups

### DISCUSSION THREAD

**Discussion by cberenguer** [PhilippeMarcMeyer/vanillaSelectBox#40] [Ideas]
Hello, I am using your library and I have the following situation, I have a select that is grouped and I want to part of looking for the options, also looking for the groups, do you have any idea that can help me, thanks
![image](https://user-images.githubusercontent.com/16119930/119895778-29fea880-bf0c-11eb-96e0-976ec043093f.png)

**Comment by PhilippeMarcMeyer** [comment]
Do you mean you want to search separatly like if you had 2 search fields [ ...... ] [ ..... ]
One to show only part of the groups as a result
another one to search the items within the found groups ?

for instance [ G1] [ Tan ] would find G15,G17,G11,G12,G19 groups (for what i can see on the picture) and finaly find Tank 2 in G11 and show only 
G11
[ ] Tank 2
?
Explain please because in my mind the goal of this tool is to allow searching options and not groups
I want to evolve it in order to allow checking/unchecking a whole group with one click but i think searching groups would be confusing to the user

**Comment by PhilippeMarcMeyer** [comment]
you didnt answered. have you tried the v 0.60 ?
